src/Controls/src/Core/Handlers/Items/Android/MauiCarouselRecyclerView.cs

```diff
@@ -104,10 +103,21 @@ public override void TearDownOldElement(CarouselView oldElement)
 			if (ItemsView != null)
 				ItemsView.Scrolled -= CarouselViewScrolled;
 
-			ClearLayoutListener();
 			base.TearDownOldElement(oldElement);
 		}
 
+		protected override void OnAttachedToWindow()
+		{
+			base.OnAttachedToWindow();
+			AddLayoutListener();
```

Comment:
I do wonder if we want this to be a layout listener. Global Layout Listeners fire for everything, so any changes to the layout anywhere causes the code inside the CV to execute. Maybe that's the intent? It does seem a little excessive.

Reply:
https://github.com/dotnet/maui/issues/18774